MVCapplication/src/main/java/com/softserve/if072/mvcapp/service/StoragePageService.java

```diff
@@ -27,14 +30,14 @@
 
     @Value("${application.restStorageURL}")
     private String storageUrl;
-
     @Value("${application.restShoppingListURL}")
     private String shoppingListURL;
 
     private final RestTemplate restTemplate;
     private final ShoppingListService shoppingListService;
     private final AnalyticsService analyticsService;
     private final MessageService messageService;
+    private final String productEnding = "storage.productEnding";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1170)